src/conventional/amdar_bufr2ioda.py

```diff
@@ -0,0 +1,506 @@
+    ("aircraft_heading", "integer", "degrees"),
+    ("latitude", "float", "degrees_north"),
+    ("longitude", "float", "degrees_east"),
+    ("height", "integer", "m"),
+    ("dateTime", "long", "seconds since 1970-01-01T00:00:00Z"),
+]
+meta_keys = [m_item[0] for m_item in locationKeyList]
+
+metaDataKeyList = {
+    'aircraft_id': ['aircraftRegistrationNumberOrOtherIdentification'],
+    'aircraft_flightNum': ['aircraftFlightNumber'],
+    'aircraft_tailNum': ['aircraftTailNumber'],
+    'obs_sequenceNum': ['observationSequenceNumber'],
+    'originationAirport': ['originationAirport'],
+    'destinationAirport': ['destinationAirport'],
+    'flight_phase': ['detailedPhaseOfFlight'],
+    'roll_angle': ['aircraftRollAngle'],
+    'roll_angle_quality': ['aircraftRollAngleQuality'],
+    'aircraft_speed': ['aircraftTrueAirspeed'],
+    'aircraft_heading': ['aircraftTrueHeading'],
+    'latitude': ['latitude'],
+    'longitude': ['longitude'],
+    'height': ['Constructed', 'globalNavigationSatelliteSystemAltitude', 'height', 'flightLevel'],
+    'dateTime': ['Constructed'],
+}
+
+# True incoming BUFR observed variables.
+raw_obsvars = ['airTemperature', 'mixingRatio', 'windDirection', 'windSpeed']
+
+# The outgoing IODA variables and their units.
+obsvars = ['air_temperature', 'specific_humidity', 'eastward_wind', 'northward_wind']
+obsvars_units = ['K', 'kg kg-1', 'm s-1', 'm s-1']
+
+VarDims = {
+    'air_temperature': ['nlocs'],
+    'specific_humidity': ['nlocs'],
+    'eastward_wind': ['nlocs'],
+    'northward_wind': ['nlocs'],
+}
+
+metaDataName = iconv.MetaDataName()
+obsValName = iconv.OvalName()
+obsErrName = iconv.OerrName()
+qcName = iconv.OqcName()
+
+AttrData = {
+    'converter': os.path.basename(__file__),
+    'ioda_version': 2,
+    'description': 'Aircraft observations converted from BUFR',
+    'source': 'LDM at NCAR-RAL',
+    'source_files': ''
+}
+
+DimDict = {
+}
+
+float_missing_value = nc.default_fillvals['f4']
+int_missing_value = nc.default_fillvals['i4']
+double_missing_value = nc.default_fillvals['f8']
+long_missing_value = nc.default_fillvals['i8']
+string_missing_value = '_'
+bufr_missing_value =  CODES_MISSING_LONG
+
+iso8601_string = locationKeyList[meta_keys.index('dateTime')][2]
+epoch = datetime.fromisoformat(iso8601_string[14:-1])
+
+
+def main(file_names, output_file):
+
+    # initialize
+    count = [0, 0, 0]
+    start_pos = None
+    start_time = time.time()
+
+    obs_data = {}          # The final outputs.
+    data = {}              # Before assigning the output types into the above.
+    data['eastward_wind'] = []
+    data['northward_wind'] = []
+    data['specific_humidity'] = []
+    data['air_temperature'] = []
+    for key in meta_keys:
+        data[key] = []
+
+    varDict = defaultdict(lambda: DefaultOrderedDict(dict))
+    varAttrs = DefaultOrderedDict(lambda: DefaultOrderedDict(dict))
+
+    for fname in file_names:
+        logging.debug("INFO: Reading file: " + fname)
```

Comment:
Is it necessary to include `INFO`? I thought the `logging` module would handle that depending on which stream is called (debug, info, etc.)?